eventstore/views.py

```diff
@@ -20,6 +27,89 @@
     PrebirthRegistrationSerializer,
     PublicRegistrationSerializer,
 )
+from ndoh_hub.utils import validate_signature
+
+
+class MessagesViewSet(GenericViewSet):
+    def create(self, request):
+        validate_signature(request)
+        try:
+            webhook_type = request.META.get("HTTP_X_TURN_HOOK_SUBSCRIPTION", "whatsapp")
```

Comment:
We should probably use `request.headers` here as well, so that we're consistent and not using `request.headers` in some places, but `request.META` in others.

Reply:
done